src/btree/bt_split.c

```diff
@@ -1031,7 +1038,8 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref,
 	 * Add it to the session discard list, to be freed when it's safe.
 	 */
 	size = sizeof(WT_PAGE_INDEX) + pindex->entries * sizeof(WT_REF *);
-	WT_TRET(__split_safe_free(session, split_gen, exclusive, pindex, size));
+	WT_TRET(__split_safe_free(session,
+		split_gen, LF_ISSET(WT_EVICT_EXCLUSIVE), pindex, size));
```

Comment:
Similarly - the indenting looks wrong here. I suspect you used a tab, it should be 4 spaces. If the ```session``` parameter fits on the second line it should be moved there.